.travis.yml

```diff
@@ -37,7 +37,10 @@ jobs:
       name: Create Documentation
       install: docker pull squidfunk/mkdocs-material:3.0.4
       before_script: documentation/bin/createDocu.sh
-      script: docker run --rm -it -v ${TRAVIS_BUILD_DIR}/documentation:/docs squidfunk/mkdocs-material:3.0.4 build --clean --strict
+      script:
+        - docker run -u `id -u`:`id -g` --rm -it -v ${TRAVIS_BUILD_DIR}/documentation:/docs squidfunk/mkdocs-material:3.0.4 build --clean --strict
```

Comment:
we start docker now explicitly with the current user id and group id. Otherwise we run with root and and directory created by that step (especially `documentation/docs-gen` is not writable afterwards. Hence the subsequent `mkdir` fails.